grails-app/views/wps/jobComplete.gsp

```diff
@@ -0,0 +1,11 @@
+Hi,
+
+Your IMOS download is ready. You can download it by clicking here: ${jobReportUrl}
```

Comment:
This may need to change. That is because you can't tell the user the data is ready as it might have failed. So need to tell the user the job has completed and she can click the URL to get more information and potentially also download the data.